tests/test_basic.py

```diff
@@ -1,11 +1,19 @@
+import unittest
+import os.path
+import sys
+import argparse
+
+sys.path = [os.path.join(os.path.dirname(os.path.realpath(__file__)),'..')]+sys.path
 from build_manpages.manpage import Manpage
 
-import argparse
 
-class TestEscapes(object):
+class TestEscapes(unittest.TestCase):
```

Comment:
Why this change?

Reply:
because otherwise python tests/test_examples.py doesn't discover it.

Reply:
Interesting, I never had those problems...  But it doesn't matter, the build seems to succeed with your changes.